ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -220,7 +220,7 @@ do
     break
   fi
 
-  non_imported_imagestreams=$(oc -n openshift get imagestreams -o go-template='{{range .items}}{{$namespace := .metadata.namespace}}{{$name := .metadata.name}}{{range .status.tags}}{{if not .items}}{{$namespace}}/{{$name}}:{{.tag}}{{"\n"}}{{end}}{{end}}{{end}}' | (grep -v 'apicast\|jboss\|jenkins\|openjdk\|redhat-sso' || true))
+  non_imported_imagestreams=$(oc -n openshift get is -o go-template='{{range .items}}{{$namespace := .metadata.namespace}}{{$name := .metadata.name}}{{range .status.tags}}{{if not .items}}{{$namespace}}/{{$name}}:{{.tag}}{{"\n"}}{{end}}{{end}}{{end}}')
```

Comment:
I think it's probably worth keeping `imagestreams`.  In [KaaS][1] testing, that one worked while `is` gave me some kind of lookup error.  Otherwise looks good to me.

[1]: https://kaas-kaas.apps.ota-stage.q2z4.p1.openshiftapps.com/